src/nvim/CMakeLists.txt

```diff
@@ -230,6 +230,11 @@ if(WIN32)
   endif()
 elseif(APPLE)
   target_link_libraries(nvim PRIVATE "-framework CoreServices")
+
+  # Really export symbols - symbols may not be visible even though
+  # ENABLE_EXPORTS is set to true. See
+  # https://github.com/kevinhwang91/nvim-ufo/issues/162.
+  target_link_libraries(nvim PRIVATE -Wl,-export_dynamic)
```

Comment:
Personally I'd use the [`LINK_FLAGS` property](https://cmake.org/cmake/help/latest/prop_tgt/LINK_FLAGS.html) for this (see also recommendation in the [docs for `target_link_libraries`](https://cmake.org/cmake/help/latest/command/target_link_libraries.html)), but this also works I guess.

[`target_link_options`](https://cmake.org/cmake/help/latest/command/target_link_options.html#command:target_link_options) seems to be the recommended way to do it, but it requires CMake 3.13+.